test/azure-cli/scenarios.json

```diff
@@ -38,5 +38,15 @@
         "installUsingPython": true
       }
     }
+  },
+  "install_with_python_3_12_bookworm": {
+    "image": "mcr.microsoft.com/devcontainers/python:1-3.12-bookworm",
+    "user": "vscode",
+    "features": {
+      "azure-cli": {
+        "version": "latest",
+        "installUsingPython": true
```

Comment:
Can we add another scenario to reflect the use case in https://github.com/devcontainers/features/issues/899 ? ie with `installUsingPython: false`. (Else, you could also add `"azure-cli": {}` to the `features` object.